sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/models/EventHubConsumerOptions.java

```diff
@@ -202,6 +222,17 @@ public int getPrefetchCount() {
         return prefetchCount;
     }
 
+    /**
+     * Gets whether or not the consumer should request information on the last enqueued event on its associated
+     * partition, and track that information as events are received.
+     *
+     * @return {@code true} if the resulting consumer will keep track of the last enqueued information for that
+     *     partition; {@code false} otherwise.
+     */
+    public boolean getLastEnqueuedEventProperties() {
```

Comment:
For auto-generated methods that return `boolean`, `get` is an acceptable prefix but in this case, should it be `isLastEnqueuedEventPropertiesTracked` to match `trackLastEnqueuedEventProperties`?

Reply:
Thanks for the catch. I missed that.